src/js/flights/flight.js

```diff
@@ -0,0 +1,60 @@
+import getPlace from './places';
+import flightData from './flightData';
+
+const getFlight = async (placeID, departureDate) => {
+  return await fetch(
```

Comment:
omg a co to za konstrukcja? :eyes:
albo async await z try catchem albo promise z then I catchem, nie miksujemy tych rzeczy :eyes: